services/graphql/cmd/graphql/main.go

```diff
@@ -6,24 +6,39 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/graph-gophers/graphql-go"
-	"github.com/graph-gophers/graphql-go/relay"
-	"go.kamp.us/services/graphql/resolver"
-	"go.kamp.us/services/graphql/schema"
+	// "os"
 
-	pano_api "go.kamp.us/services/pano-api/rpc/pano-api"
+	graphql "github.com/graph-gophers/graphql-go"
+	"github.com/graph-gophers/graphql-go/relay"
+	// "go.kamp.us/services/graphql/resolver"
+	// "go.kamp.us/services/graphql/schema"
+	// pano_api "go.kamp.us/services/pano-api/rpc/pano-api"
 )
 
+type query struct{}
+
+func (_ *query) Hello() string { return "Hello, world!" }
+
 func main() {
-	panoapiClient := pano_api.NewPanoAPIProtobufClient(os.Getenv("PANOAPI_URI"), &http.Client{})
+	// panoapiClient := pano_api.NewPanoAPIProtobufClient(os.Getenv("PANOAPI_URI"), &http.Client{})
 
-	clients := resolver.Clients{
-		PanoAPI: panoapiClient,
-	}
+	// clients := resolver.Clients{
+	// 	PanoAPI: panoapiClient,
+	// }
+
+	// schema := graphql.MustParseSchema(schema.Schema, &resolver.Resolver{Clients: &clients}, graphql.UseStringDescriptions())
+
+	s := `
+    type Query {
```

Comment:
it would be better to read a graphql schema file externally (filesystem) or with `go:embed` so that we can standardize the schema syntax

Reply:
Yep, we already have that, i've just made it so that we have a simple binary to deploy. we will deploy pano first, then we'll use the real schema under https://github.com/kamp-us/monorepo/blob/dev/services/graphql/schema/schema.graphql